Cargo.toml

```diff
@@ -34,7 +34,8 @@ members = [
     "python_binding",
     "auth",
     "config",
-    "examples"
+    "examples",
+    "praveglet"
```

Comment:
We need a better name. 'praveglet' sounds like it would be a pravega kubelet.

Reply:
done.